credentials/simple-vc-v2/credential.json

```diff
@@ -0,0 +1,13 @@
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2"
+  ],
+  "id": "http://vc.example.com/credentials/4643",
```

Comment:
I can't remember what / how IDs work or interface with the VC playground. Should we just remove this?

```suggestion
```

Reply:
We don't need it there...so no reason not to remove it. I was just basing it off what was in the VCDM spec as closely as possible...but that's not necessary.